src/Label/Events/CopyCreated.php

```diff
@@ -19,9 +19,10 @@ public function __construct(
         string $name,
         Visibility $visibility,
         Privacy $privacy,
-        UUID $parentUuid
+        UUID $parentUuid,
+        bool $excluded = false
```

Comment:
I really wonder if we need that parameter here. What is the purpose of creating an excluded label from the start, it should never be created.